src/Sentry.Extensions.Protobuf/Sentry.Extensions.Protobuf.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.0;net5.0</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Google.Protobuf" Version="3.13.0" />
```

Comment:
Would it make sense to bundle the GPRC and Protobuf stuff into a single package? Sentry.Grpc?

I don't think we'll need (anytime soon) a Protobuf package stand alone and it would simplify things on. our end.

Reply:
Yep.